src/enum.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+export default {
+  // Finds the element at the given index (zero-based).
+  // Returns default if index is out of bounds.
+  // A negative index can be passed, which means the enumerable is enumerated once
+  // and the index is counted from the end (e.g. -1 finds the last element).
+  at(array: Array<any>, index: number, defaultValue: any = null): any {
+    const { length } = array;
+    if (index < 0) {
+      const reverseIndex = length + index;
+      if (reverseIndex > 0 && reverseIndex < length) {
+        return array[reverseIndex];
+      }
+    } else if (index < length) {
+      return array[index];
+    }
+    return defaultValue;
+  },
```

Comment:
This one started getting pretty `if`y to mimic the negative index behavior of Elixir's `Enum.at/3`, though I could just be failing to see a simpler implementation.

Reply:
Hey as long as it works 😄   Have you seen the Lo-dash function `_.nth`? Since lodash is a dep it may be an option... unsure if it has the same functionality that you're using though.
https://lodash.com/docs/4.17.5#nth